components/BankTransferStep.tsx

```diff
@@ -32,9 +32,9 @@ export const BankTransferStep = () => {
       </table>
 
       <p>
-        Du kan opsætte månedlige overførsler i din netbank eller mobilbank med
-        den samme besked. Så sender vi donationen til det valgte velgørende
-        formål og indberetter evt. fradrag.
+        Du kan lavere en eller flere overførsler med ovenstående besked i din netbank eller mobilbank.
+        Så sender vi donationen til det valgte velgørende formål og indberetter evt. fradrag.
+        Vi indrapporterer kun det, der faktisk er overført. Du vælger altså selv beløb og tidspunkt(er).
```

Comment:
Maybe we shouldn't so "willingly" offer people to change amount from what they've just selected in the previous step? We sure _support_ that, but when amount changes, what basically happens is that I fill out the form for them with a new amount... 😅 On the other hand, this (almost) never happened yet, people do follow what they selected on the form, or fill it out again, so maybe I'm over thinking it...

Reply:
Just wait till you see me donating ;-) 

OK, yes, I agree it's better to remove that last sentence "Du vælger altså selv beløb og tidspunkt(er)".